src/zcbenchmarks.cpp

```diff
@@ -31,6 +31,31 @@
 #include "zcash/IncrementalMerkleTree.hpp"
 
 using namespace libzcash;
+// This method is based on Shutdown from init.cpp
+void shutdown_wallet()
+{
+    LogPrintf("%s: In progress...\n", __func__);
+    static CCriticalSection cs_Shutdown;
+    TRY_LOCK(cs_Shutdown, lockShutdown);
+    if (!lockShutdown)
+        return;
+
+if (pwalletMain)
+    pwalletMain->Flush(false);
+#ifdef ENABLE_MINING
+    GenerateBitcoins(false, NULL, 0);
```

Comment:
This should be re-enabled after loading the wallet (like how it is similarly enabled in `init.cpp`).

Reply:
don't follow

Reply:
See here: https://github.com/zcash/zcash/blob/master/src/init.cpp#L1679-L1687

Reply:
You mean you want to replace the generatebitcoins calls by this? 
`if (pwalletMain || !GetArg("-mineraddress", "").empty())
        GenerateBitcoins(GetBoolArg("-gen", false), pwalletMain, GetArg("-genproclimit", 1));`